### ISSUE TITLE
ESS took 2 minutes to start when CSS is unreachable

### ISSUE THREAD

**Issue by linggao** [open-horizon/anax#963]
The following call took more than 2 minutes to finish on an ibm cloud VM with 2 cpus and 4 GB of ram.
````
import “github.com/open-horizon/edge-sync-service/core/base”
base.Start("", true)
````
It prevents the edge node from doing any ESS related work until it has completed starting. It also prevents the node from completing unregistration until ESS has completed starting. Please reduce the ESS starting time.

**Comment by shmuelk** [comment]
Can you please send me the log?

**Comment by shmuelk** [comment]
@bmpotter Why are you starting the ESS if you are trying to unregister a node? In such a case is there enough configuration information to properly start the ESS?

**Comment by bmpotter** [comment]
Shmuel,

The scenario is:
1. The horizon agent (including the embedded ess) is installed on the edge node
1. I register the edge node to horizon to use a pattern using `hzn register ...`. This starts an ess worker in the agent, and also starts my edge service.
1. The edge service starts and i see it is running successfully
1. Right after that i use `hzn unregister` to stop my edge service

This last step, it takes more than 2 minutes on my edge node, and ling determined that it was because it was waiting for the ess worker to complete its initialization before it could complete the unregister process.

I've confirmed that this delay only occurs when ess can't successfully communicate with the css. We had some ingress rules wrong for the css. Once we fixed those, the delay went away. But the ess worker should still be able to be shutdown in a timely manner even if it can't communicate with the css. Our architecture specifies that the edge nodes don't need to have constant connectivity to the internet.
